creator-node/src/fileManager.js

```diff
@@ -295,17 +297,36 @@ async function saveFileForMultihashToFS (serviceRegistry, logger, multihash, exp
     }
 
     // verify that the contents of the file match the file's cid
+    let fileData = null
+    let fileSize = null
+    try {
+      fileData = (await fs.readFile(expectedStoragePath)).toString().slice(0, 99)
+      fileSize = (await fs.stat(expectedStoragePath)).size
+    } catch (e) {
+      logger.warn('Could not get fs statistics for path', expectedStoragePath)
```

Comment:
you're right, i can't remember why i put this in a separate try catch. i think at first it was for information but since we use it for the verification flow it should be in the main path